test/support/test_helper_methods.rb

```diff
@@ -0,0 +1,7 @@
+module TestHelperMethods
+  def log_in_user(user)
+    fill_in 'session[username]', with: user.username
+    fill_in 'session[password]', with: user.password
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.